index.html

```diff
@@ -1,17 +1,160 @@
+          change the world through fostering a love of learning, an appreciation
+          of diversity and a deep understanding of themselves and others.
+        </p>
+      </div>
+      <img src="./images/teacher-kids.jpg" class="head-img" alt="" />
+      <div id="date-location">
+        <h4 id="date">PTA 2022.05.24(TUE) ~ 11AM(UTC)</h4>
+        <p id="location">@Quami City, Ghana</p>
+      </div>
+    </section>
+    <!-- Stats Section -->
+    <section id="stats">
+      <h2 id="stats-heading">Stats</h2>
+      <div id="stats-grid">
+        <!-- Card One -->
+        <div class="stats-card">
+          <div class="icon">
+            <i class="fa-solid fa-chalkboard-user"></i>
+          </div>
+          <div class="stats-title">
+            <h4>Teachers</h4>
+          </div>
+          <div class="stats-description">
+            <p>We have trained proffessionals who are mostly examiners.</p>
+          </div>
+        </div>
+        <!-- Card Two -->
+        <div class="stats-card">
+          <div class="icon">
+            <i class="fa-solid fa-school"></i>
+          </div>
+          <div class="stats-title">
+            <h4>Campuses</h4>
+          </div>
+          <div class="stats-description">
+            <p>we have more than 20 Campuses in West Africa alone</p>
+          </div>
+        </div>
+        <!-- Card Three -->
+        <div class="stats-card">
+          <div class="icon">
+            <i class="fa-solid fa-graduation-cap"></i>
+          </div>
+          <div class="stats-title">
+            <h4>Courses</h4>
+          </div>
+          <div class="stats-description">
+            <p>
+              we offer 18 cruciual programs. Each student gets to find their
+              own.
+            </p>
+          </div>
+        </div>
+        <!-- Card Four -->
+        <div class="stats-card">
+          <div class="icon">
+            <i class="fa-solid fa-people-group"></i>
+          </div>
+          <div class="stats-title">
+            <h4>Students</h4>
+          </div>
+          <div class="stats-description">
+            <p>we've trained over 10,000 students accross all our brances.</p>
+          </div>
+        </div>
+        <!-- Card Five -->
+        <div class="stats-card">
+          <div class="icon">
+            <i class="fa-solid fa-earth-africa"></i>
+          </div>
+          <div class="stats-title">
+            <h4>Countries</h4>
+          </div>
+          <div class="stats-description">
+            <p>we are in over 10 countries worldwide.</p>
+          </div>
+        </div>
+      </div>
+      <button type="button" id="join-button" class="bubbly-button">
+        Register Your Child
+      </button>
+    </section>
+    <!--Teachers Section  -->
+    <section id="teacher-section">
+      <h2 class="section-intro">Our Teachers</h2>
+      <div id="teacher-grid">
+         
+      </div>
```

Comment:
- [ ] you are missing this small image beside the image so kindly fix it
### Original design
![hhh](https://user-images.githubusercontent.com/81727988/169627456-c8c43757-5f4d-40bb-8007-c613224f49d5.png)